torch/optim/adam.py

```diff
@@ -150,10 +104,8 @@ def _init_group(
                     raise RuntimeError('`requires_grad` is not supported for `step` in differentiable mode')
                 state_steps.append(state['step'])
 
-        return grad_scale, found_inf
-
     @_use_grad_for_differentiable
-    def step(self, closure=None, *, grad_scaler=None):
+    def step(self, closure=None):
```

Comment:
fwiw the kwarg of `grad_scaler` is in v1.13.1 so this change would be backward breaking in a sense

Reply:
Thanks for raising this. That sounds fine to me as almost no one is using it: https://github.com/search?q=%2Fstep%5C%28grad_scaler%3D.*%5C%29%2F&type=code
cc @wearepal who is the only reference that explicitly uses it